### PR TITLE
Remove component documents

### PR DESCRIPTION
**Pull request by mmitoraj**
**Description**

Changes proposed in this pull request:

- Cleaned up components' leftovers

**Related issue(s)**
#18455

### CODE REVIEW

File: docs/04-operation-guides/operations/10-backup-kyma.md [hunk 1]
```diff
@@ -1,206 +0,0 @@
----
-title: Back up Kyma
```
Comment (a-thaler) [inline docs/04-operation-guides/operations/10-backup-kyma.md:2]: Is that document covered somewhere?

File: docs/04-operation-guides/operations/obsv-01-access-logs.md [hunk 2]
```diff
@@ -1,23 +0,0 @@
----
-title: Access Kyma application logs
```
Comment (a-thaler) [inline docs/04-operation-guides/operations/obsv-01-access-logs.md:2]: Is that document covered somewhere?

**Review by a-thaler — CHANGES_REQUESTED** [review]

File: docs/04-operation-guides/operations/10-backup-kyma.md (hunk 1 above)
Comment (mmitoraj) [reply docs/04-operation-guides/operations/10-backup-kyma.md:2]: We have that general document -> https://help.sap.com/docs/btp/sap-business-technology-platform/kyma-environment-backup?version=Cloud. Let's enrich it with the details we want to remove. I'm leaving the doc for the time being. [Issue](https://github.tools.sap/kyma/backlog/issues/4813) created.

File: docs/04-operation-guides/operations/obsv-01-access-logs.md (hunk 2 above)
Comment (mmitoraj) [reply docs/04-operation-guides/operations/obsv-01-access-logs.md:2]: Issue created to add the content to the HP -> https://github.tools.sap/kyma/backlog/issues/4814

**Review by a-thaler — APPROVED** [review]

File: docs/02-get-started/01-quick-install.md [hunk 3]
```diff
@@ -1,65 +1,86 @@
...
-When asked whether to install the Kyma certificate, confirm.
+  ```bash
+  kyma provision k3d
+  ```
 
-> **NOTE:** Check out [more installation options for Kyma](../04-operation-guides/operations/02-install-kyma.md).
+  When asked whether to install the Kyma certificate, confirm.
 
-### Verify the installation
+2. Choose a module and deploy its module manager and apply the module configuration. The operation installs a Kyma module of your choice on a Kubernetes cluster. See the already available Kyma modules with their quick installation steps and links to their GitHub repositories:
```
Comment (grego952) [inline docs/02-get-started/01-quick-install.md:23]:
```suggestion
2. Choose a module, deploy its module manager, and apply the module configuration. The operation installs a Kyma module of your choice on a Kubernetes cluster. See the already available Kyma modules with their quick installation steps and links to their GitHub repositories:
```

**Review by grego952 — APPROVED** [review]

**Pull request merged**